examples/instruct_pix2pix/train_instruct_pix2pix_sdxl.py

```diff
@@ -878,7 +890,7 @@ def collate_fn(examples):
     if args.pretrained_vae_model_name_or_path is not None:
         vae.to(accelerator.device, dtype=weight_dtype)
```

Comment:
wasn't sure if we want to keep the original value here or just clean it up and keep it all consistent via vae_precision